dashboard/test/ui/features/gdpr_dialog.feature

```diff
@@ -0,0 +1,36 @@
+@dashboard_db_access
+
+Feature: GDPR Dialog - data transfer agreement
+
+  Scenario: EU user sees the GDPR Dialog on dashboard, opt out
+    Given I am a teacher
+    Given I am on "http://studio.code.org/home?force_in_eu=1"
+    Then I wait to see ".ui-test-gdpr-dialog"
+    Then I wait to see ".ui-test-gdpr-dialog-logout"
+    Then I click selector ".ui-test-gdpr-dialog-logout"
+    Then check that I am on "http://code.org/"
+
+  Scenario: EU user sees the GDPR Dialog on dashboard, opt in
+    Given I am a teacher
+    Given I am on "http://studio.code.org/home?force_in_eu=1"
+    Then I wait to see ".ui-test-gdpr-dialog"
+    Then I click selector ".ui-test-gdpr-dialog-accept"
+    # Confirm dialog closed and I can do something on this page
+    And I create a new section
+
+  Scenario: GDPR Dialog privacy link works
+    Given I am a teacher
+    Given I am on "http://studio.code.org/home?force_in_eu=1"
+    Then I wait to see ".ui-test-gdpr-dialog"
+    Then I click selector ".ui-test-gdpr-dialog-privacy-link"
+    Then check that I am on "http://code.org/privacy"
```

Comment:
Funnily, today I made all the "See the privacy policy" links open in a new tab.  Would it be worth doing same here?

Actually, if an EU user clicked the privacy policy link here, wouldn't the /privacy page be covered by the dialog too?

Reply:
@breville  Do I need to do that (open privacy policy in a new tap) on the accounts page?

Reply:
Good question, and @tanyaparker should chime in.  I generally open in a new tab when it feels like a navigational dead-end and the user will most likely want to still have a tab open where they were, and doubly so when they have likely already filled out a bunch of a form they were about to submit.  So I'd personally vote for yes if you match this criteria.

Reply:
Ordinarily if we link to one of our own pages, it would not open in a new tab. We generally open in a new tab for external links. But in this case, especially since they're filling out a form (seconding Brendan) I wouldn't want to direct them away form the form and wouldn't want them to possibly lose what data they had already completed. I think all of these `See privacy policy` links are justified to open in a new tab.